### PR TITLE
release/P7b: set up P7b test case

### PR DESCRIPTION
**Pull request by junwang-noaa**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

This PR is to add ccpp p7b suite file and update regression test script for the atmosphere physics namelist/input data in order to run P7b.
1) add suite_FV3_GFS_v16_coupled_nsstNoahmpUGWPv1.xml suite file
2) Update regression test script to update the input files to run the P7b test

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes ufs-weather-model issue [#637](https://github.com/ufs-community/ufs-weather-model/issues/637)

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [ ] hera.intel

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

- noaa-emc/fv3atm [PR#329](https://github.com/NOAA-EMC/fv3atm/pull/329)
- ufs-weather-model [PR#636](https://github.com/ufs-community/ufs-weather-model/pull/636)

### CODE REVIEW

File: tests/fv3_conf/cpld_bmark_tiled_run.IN [hunk 1]
```diff
@@ -109,3 +109,22 @@ if [ $WRITE_DOPOST = .true. ]; then
...
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_OC.v1_3.dat  optics_OC.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_DU.v15_3.dat optics_DU.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SS.v3_3.dat  optics_SS.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SU.v1_3.dat  optics_SU.dat
+fi
+
+#ugwpv1
+if [ $DO_UGWP_V1 = .true. ]; then
+  cp    @[INPUTDATA_ROOT]/FV3_input_data/ugwp_c384_tau.nc ./ugwp_limb_tau.nc
+  cp    @[INPUTDATA_ROOT]/FV3_input_datac384/INPUT_L127/oro_data_ls* ./INPUT
```
Comment (DeniseWorthen) [inline tests/fv3_conf/cpld_bmark_tiled_run.IN]: So these oro_data_ls* files are meant to optionally over-write the oro_data.tileX.nc files which are copied in at LN 34-40?

**Comment by junwang-noaa** [comment]
No, they are additional files needed for the gwd_opt option used by ugwdv1.

File: tests/tests/cpld_bmark_wave_v16_p7b_35d [hunk 2]
```diff
@@ -0,0 +1,163 @@
+#
```
Comment (DeniseWorthen) [inline tests/tests/cpld_bmark_wave_v16_p7b_35d:1]: We'll need to add this 35d test to the rt_35d.conf along w/ adding the suite to the compile line there. But you may still be making changes/testing things?

Edit: I re-read your email so now I understand you are waiting for the input data at C384.

**Comment by junwang-noaa** [comment]
Yes, I am first testing the cpld_bmark_wave_v16_p7b, once it works I will
add the related changes in rt_35d.conf

File: tests/tests/cpld_bmark_wave_v16_p7b_35d (hunk 2 above)
Comment (junwang-noaa) [reply tests/tests/cpld_bmark_wave_v16_p7b_35d:1]: @DeniseWorthen the 35d test is tested and added to rt_35d.conf. Please review the PR.

**Review by DeniseWorthen — APPROVED** [review]
run scripts look good

**Comment by junwang-noaa** [comment]
@AvichalMehra-NOAA @yangfanglin The P7b branch has all the required features specified in the prototype documents.

**Comment by AvichalMehra-NOAA** [comment]
@junwang-noaa Thanks. We need to verify that all looks good with P7a. 
We should be ready to start running P7b in a couple of days.

**Pull request merged**